unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SocialAnalytics/SocialAnalytics.cs

```diff
@@ -0,0 +1,270 @@
+using DCL.Interface;
+using System.Collections.Generic;
+
+namespace SocialFeaturesAnalytics
+{
+    public class SocialAnalytics : ISocialAnalytics
+    {
+        private const string PLAYER_MUTED = "user_muted";
+        private const string PLAYER_UNMUTED = "user_unmuted";
+        private const string VOICE_MESSAGE_STARTED_BY_FIRST_TIME = "voice_chat_start_recording";
+        private const string VOICE_MESSAGE_SENT = "voice_chat_sent_recording";
+        private const string CHANNEL_MESSAGE_SENT = "send_chat_message";
+        private const string CHANNEL_MESSAGE_RECEIVED = "chat_message_received";
+        private const string DIRECT_MESSAGE_SENT = "send_direct_message";
+        private const string DIRECT_MESSAGE_RECEIVED = "direct_message_received";
+        private const string FRIEND_REQUEST_SENT = "friend_request_sent";
+        private const string FRIEND_REQUEST_APPROVED = "friend_request_approved";
+        private const string FRIEND_REQUEST_REJECTED = "friend_request_rejected";
+        private const string FRIEND_REQUEST_CANCELLED = "friend_request_cancelled";
+        private const string FRIEND_DELETED = "friend_deleted";
+        private const string PASSPORT_OPENED = "passport_opened";
+        private const string PASSPORT_CLOSED = "passport_closed";
+        private const string PLAYER_BLOCKED = "user_blocked";
+        private const string PLAYER_UNBLOCKED = "user_unblocked";
+        private const string PLAYER_REPORT = "player_report";
+        private const string PLAYER_JOIN = "player_join";
+        private const string PLAY_EMOTE = "used_emote";
+
+        public void SendPlayerMuted(string toUserId)
+        {
+            PlayerType? toPlayerType = GetPlayerTypeByUserId(toUserId);
+
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("to", toPlayerType.ToString());
+
+            GenericAnalytics.SendAnalytic(PLAYER_MUTED, data);
+        }
+
+        public void SendPlayerUnmuted(string toUserId)
+        {
+            PlayerType? toPlayerType = GetPlayerTypeByUserId(toUserId);
+
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("to", toPlayerType.ToString());
```

Comment:
`toPlayerType` may be null.. perhaps null checking as a method pre-condition

Reply:
In case of `toPlayerType` is null, we shouldn't send the analytic. I missed do this check. Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/678828e6b8345a710e289d0af3ae46fb12273a26

Reply:
This is intentional; if `GetPlayerTypeByUserId()` return null, then `toPlayerType` will be assigned as null and we want to send this info as so in the metric.

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/678828e6b8345a710e289d0af3ae46fb12273a26